decidim-proposals/spec/system/proposal_show_spec.rb

```diff
@@ -52,6 +52,25 @@ def visit_proposal
           end
         end
       end
+
+      describe "author tooltip" do
```

Comment:
I think we can just change this to `describe "when participant is deleted"` and remove the context, right?